images/build/Dockerfiles/full.Dockerfile

```diff
@@ -21,8 +21,10 @@ RUN ./build.sh golang     /opt/startupcmdgen/golang
 FROM buildpack-deps:${DEBIAN_FLAVOR}-curl
 ARG DEBIAN_FLAVOR
 ARG SDK_STORAGE_BASE_URL_VALUE="https://oryx-cdn.microsoft.io"
+ARG AI_CONNECTION_STRING
```

Comment:
nit: I'd sync with William to see if there's a reason why the instrumentation key was never included in this image just in case we don't want to have telemetry here for some reason